pkg/volume/glusterfs/glusterfs.go

```diff
@@ -66,6 +66,7 @@ const (
 	annGlusterSecretNamespace = "glusterfs.kubernetes.io/secretnamespace"
 	annGlusterUserKey         = "glusterfs.kubernetes.io/userkey"
 	annGlusterUser            = "glusterfs.kubernetes.io/userid"
+	annGlusterClusterId       = "glusterfs.kubernetes.io/clusterid"
```

Comment:
This annotation is not used anywhere, please remove it.
[Also, if you like, you can trim the other unused annotations too]